ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -13,8 +13,59 @@
 
 package types
 
+import "fmt"
+
 const (
-	BlackHolePortFaultType = "network-blackhole-port"
-	LatencyFaultType       = "network-latency"
-	PacketLossFaultType    = "network-packet-loss"
+	BlackHolePortFaultType    = "network-blackhole-port"
+	LatencyFaultType          = "network-latency"
+	PacketLossFaultType       = "network-packet-loss"
+	missingRequiredFieldError = "required parameter %s is missing"
 )
+
+type NetworkFaultRequest interface {
+	ValidateRequest() error
+	ToString() (string, error)
+}
+
+type NetworkBlackholePortRequest struct {
+	Port        *int    `json:"Port"`
+	Protocol    *string `json:"Protocol"`
+	TrafficType *string `json:"TrafficType"`
+}
+
+type NetworkFaultInjectionResponse struct {
+	Status string `json:"status,omitempty"`
```

Comment:
Will the `status` in the json be showing as `status`? If so, how about `Status` (to be consistent with the request payload)

Reply:
sure